Formula/httpd.rb

```diff
@@ -112,7 +307,7 @@ def plist; <<-EOS.undent
       end
       sleep 3
 
-      assert_match expected_output, shell_output("curl 127.0.0.1:8080")
+      assert_match expected_output, shell_output("curl -s 127.0.0.1:8080")
```

Comment:
What's this needed for?

Reply:
Just keeps the terminal output less noisy. Just a pet peeve when doing testing locally.

Reply:
Ok, cool.